Platform/AlderlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -68,6 +68,30 @@ CONST PLT_DEVICE mPlatformDevices[] = {
     .Type = OsBootDeviceNvme,
     .Instance = 1
   },
+  {
```

Comment:
Raghava, 
Can you please confirm the b/d/f for UFS? 
I think we talked about UFS being on 0:18:7 and so ISH on 0:18:0 needs to be enabled to for UFS to be enumerated.